rc/filetype/makefile.kak

```diff
@@ -33,7 +33,7 @@ add-highlighter shared/makefile/content default-region group
 add-highlighter shared/makefile/comment region '#' '$' fill comment
 add-highlighter shared/makefile/evaluate-commands region -recurse '\(' '\$\(' '\)' fill value
 
-add-highlighter shared/makefile/content/ regex ^[\w.%-]+\h*:\s 0:variable
+add-highlighter shared/makefile/content/ regex ^\S.*?(::|:|!) 0:variable
```

Comment:
I think this would make a non-target line e.g. `ABC:=1`, hence the trailing `\s` in the previous regex.